.gitignore

```diff
@@ -0,0 +1,3 @@
+/.dart_tool
```

Comment:
Are these your local files not related to the project?

Reply:
yes, my local. But we can add it to gitignore.